value.go

```diff
@@ -152,6 +153,11 @@ func (r *safeRead) Entry(reader *bufio.Reader) (*Entry, error) {
 		return nil, err
 	}
 
+	// Encounter preallocated region, just act as EOF.
+	if hbuf[0] == 0 {
```

Comment:
Please create a constant for 0 with a better name, such as EntryFlagNone.

Reply:
Added `isEncodedHeader` and `metaNotEntry`.